almanachclient/v1/client.py

```diff
@@ -27,35 +27,87 @@ def get_url(self):
         return self.url
 
     def get_info(self):
+        """Display information about the current version and counts of entities in the database.
+
+        """
         return self._get('{}/{}/info'.format(self.url, self.api_version))
 
     def get_volume_types(self):
+        """List volume types.
+
+        """
         return self._get('{}/{}/volume_types'.format(self.url, self.api_version))
 
     def get_volume_type(self, volume_type_id):
+        """Get a volume type.
+
+        :arg str volume_type_id: Volume Type Uuid
+
+        :raises ClientException
+        """
         return self._get('{}/{}/volume_type/{}'.format(self.url, self.api_version, volume_type_id))
 
     def create_volume_type(self, volume_type_id, volume_type_name):
+        """Create a volume type.
+
+        :arg str volume_type_id: The Volume Type id
+        :arg str volume_type_name: The Volume Type name
+
+        :raises ClientException
+        """
         url = '{}/{}/volume_type'.format(self.url, self.api_version)
         data = {'type_id': volume_type_id, 'type_name': volume_type_name}
         self._post(url, data)
         return True
 
     def delete_volume_type(self, volume_type_id):
+        """Delete the volume type.
+
+        :arg str volume_type_id: Volume Type Uuid
+
+        :raises ClientException
+        """
         self._delete('{}/{}/volume_type/{}'.format(self.url, self.api_version, volume_type_id))
         return True
 
     def get_volumes(self, tenant_id, start, end):
+        """List volumes for a tenant.
+
+        :arg Uuid tenant_id: Tenant Uuid
+        :arg DateTime start: Y-m-d H:M:S.f
```

Comment:
- tenant_id is a Python string not a uuid object
- You don't need to specify the date format because it's a datetime object: https://docs.python.org/3/library/datetime.html#datetime-objects